package.json

```diff
@@ -6,7 +6,7 @@
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
-    "dev-start": "source ./env/development && source ./env/dev-secrets && node --debug app.js",
+    "dev-start": "source ./env/development && source ./env/dev-secrets && node --inspect app.js",
```

Comment:
bah i tried to update this too but it wasnt working with my dev tools natively :/.  Might need more investigation before we change this (maybe i need to update chrome)